app/models/user.rb

```diff
@@ -8,6 +8,8 @@ class User < ApplicationRecord
   validates_presence_of :nickname, :first_name, :last_name
   validates_uniqueness_of :nickname
   has_one_attached :photo
+  has_many :memberships
```

Comment:
Same question

Reply:
same answer ;) - membership is the link between users and groups

Reply:
Same answer ;-)